onnxruntime/core/providers/js/operators/pad.cc

```diff
@@ -24,7 +24,7 @@ ONNX_OPERATOR_VERSIONED_KERNEL_EX(
     12,
     kJsExecutionProvider,
     (*KernelDefBuilder::Create())
-        .TypeConstraint("T", DataTypeImpl::GetTensorType<float>())
+        .TypeConstraint("T", JsepSupportedFloatTypes())
```

Comment:
For version <=10, we can use `JsepSupportedFloatTypes()`. After versions (>10), use `JsepSupportedDataTypes()`?